src/getting_started.md

```diff
@@ -38,8 +38,8 @@ Expand-Archive -Path alan-windows.zip -DestinationPath C:\windows
 If you wish to contribute to Alan, or if your operating system and/or CPU architecture do not match the above, you'll need a development environment to build Alan locally:
 
 * git (any recent version should work)
-* Node.js >=10.20.1, <14.0.0
-* Rust >=1.41.1
+* Node.js >=10.20.1
+* Rust >=1.45.0
```

Comment:
ah we should change that in the main repo README too!

Reply:
Wait, we didn't?